app/src/main/java/com/example/create_account/ListAdapter.kt

```diff
@@ -0,0 +1,58 @@
+package com.example.create_account
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Button
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import com.example.accountcreate.R
+
+class ListAdapter(private val mList: List<ListItem>) :
```

Comment:
Consider renaming the list variable 'mList' to make it more descriptive and relevant.